crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -690,6 +690,7 @@ where
     ///
     /// This will return `SYNCING` if the block was buffered successfully, and an error if an error
     /// occurred while buffering the block.
+    #[instrument(level = "trace", skip_all, target = "consensus::engine", ret)]
```

Comment:
Oh - TIL ret! Still would like a better way for our targets. Should we refactor them into static str?

Reply:
could try but idk if this even works